crank/QMEngine.py

```diff
@@ -69,18 +72,22 @@ def load_task_result_m(self, job_path=None):
         Return the Molecule object
         """
         orig_dir = os.getcwd()
-        if job_path != None:
+        if job_path is not None:
             os.chdir(job_path)
-        if self.native_opt == True:
+        if self.native_opt:
             m = self.load_native_output()
         else:
             m = self.load_geomeTRIC_output()
         os.chdir(orig_dir)
         return m
 
-    def run(self, cmd, input_files=[], output_files=[]):
+    def run(self, cmd, input_files=None, output_files=None):
         """ Execute a command locally or remotely, based on whether self.work_queue is set """
-        if self.work_queue == None:
+        if input_files is None:
```

Comment:
I'm not exactly sure why this change is needed.

Reply:
https://stackoverflow.com/questions/14247373/python-none-comparison-should-i-use-is-or

Reply:
Thanks, but my question about this part is: here 4 additional lines of code are added, to replace the default parameters `input_files=[]` to `input_files=None`, and overwrite it back.

Reply:
Oh declaring a list in the definition like that means you use the same list every time the function is invoked, effectively a global for the function. This can lead to very surprising things as your function now has state.